plugins/community/nanopb/v0.4.9/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1.10
+FROM python:3.11.10-alpine3.20 AS build
+WORKDIR /app
+RUN python -mvenv /app
+ADD /requirements.txt requirements.txt
+RUN source ./bin/activate \
+ && pip install --no-cache-dir -r requirements.txt \
+ && pip uninstall --yes pip \
+ && rm -f requirements.txt bin/activate.fish bin/activate.csh bin/Activate.ps1 \
+ && ln -sf /usr/bin/python /app/bin/python
+
+FROM gcr.io/distroless/python3-debian12:latest@sha256:55283e240bb5bd506e711a395092b61acf1fb14e63977ee24336285dd9e68fb8
```

Comment:
We should update to squash layers here.

Reply:
done in https://github.com/bufbuild/plugins/pull/1501/commits/5bdb46fdb62e7bd0b954250175491c893e008189